modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/FrameworkRuns.java

```diff
@@ -242,7 +242,28 @@ private boolean storeRun(String runName, SubmitRunRequest runRequest) throws Dyn
         } else {
             otherRunProperties.put(runPropertyPrefix + ".group", UUID.randomUUID().toString());
         }
-        otherRunProperties.put(runPropertyPrefix + ".requestor", requestor.toLowerCase());
+        
+        //Putting this in as a feature flag, checking if CPS property is set
+        //If set to true then it will allow it to have mixed characters, for e.g,
+        // Galasadelivery@ibm.com. Else it would set everything to lowercase by default
+
+        boolean isMixedCaseRequestorAllowed = false;
+        try {
+            String propValue ;
+            propValue = cps.getProperty("runs.requestor", "is.mixed.case");
+            if ("true".equals(propValue)) {
+                isMixedCaseRequestorAllowed = true;
+            }
```

Comment:
`Boolean.parseBoolean()` could be used to handle inputs like `True`, `TRUE`, and `tRuE`.

Since this is a feature flag, I'm happy to keep as is.